terraform/docs/applying-terraform.md

```diff
@@ -7,60 +7,49 @@ The EKS cluster is deployed via Terraform in two stages. See [adr-3] for backgro
+   either use a dummy value or look at setting up the [CDN service](../../docs/setting-up-content-delivery-network.md)
+2. `cluster-services` deployment requires some [prerequisite secrets](../../docs/prerequisite-secrets.md)
 which are not generated automatically. Create these secrets before running
-`terraform apply` for the first time.
+the Terraform apply for the first time.
 
 ## Deployment
 
-**We no longer have any deployment automation for Terraform** (since the demise of Big Concourse).
-
-For testing before merging to `main`, we can run Terraform locally against the test account.
+All terraform modules in this repository are now deployed via Terraform Cloud.
+To test changes before merging into main, open a PR and a plan will automatically
+start for your branch.
 
 When turning up from scratch, deploy the root modules in this order:
 
-1. [`terraform-lock`](../deployments/terraform-lock): creates the
-   DynamoDB table which Terraform uses to control concurrent access to the
-   state files in S3.
-1. [`ecr`](../deployments/ecr) (test and production accounts only): creates the ECR container registry from
+1. [`tfc-bootstrap`](../deployments/tfc-bootstrap): bootstraps Terraform Cloud and creates `tfc-configuration`
+workspace which is used to manage the other Terraform module workspaces.
+2. [`tfc-configuration`](../deployments/tfc-configuration): creates TFC workspaces for each terraform module and environment.
+3. [`ecr`](../deployments/ecr) (test and production accounts only): creates the ECR container registry from
    which the cluster pull container images. There is a single registry for all
    of the environments (to avoid consistency problems with image tags and
    having to copy images between registries), so this module is not deployed
    per-environment.
-1. [`cluster-infrastructure`](../deployments/cluster-infrastructure): creates the AWS resources for the cluster.
-1. Delete the `aws-auth` configmap by running `gds aws govuk-${ENV?}-admin -- aws eks update-kubeconfig --name govuk && kubectl -n kube-system delete cm aws-auth`. This is a workaround for the problem that one of the AWS-managed EKS addons creates a default aws-auth configmap which then either needs to be imported into Terraform or deleted.
-1. [`govuk-publishing-infrastructure`](../deployments/govuk-publishing-infrastructure): creates AWS resources specific to the GOV.UK apps where we are not yet
+4. [`cluster-infrastructure`](../deployments/cluster-infrastructure): creates the AWS resources for the cluster.
+5. Delete the `aws-auth` configmap by running `gds aws govuk-${ENV?}-admin -- aws eks update-kubeconfig --name govuk && kubectl -n kube-system delete cm aws-auth`. This is a workaround for the problem that one of the AWS-managed EKS addons creates a default aws-auth configmap which then either needs to be imported into Terraform or deleted.
```

Comment:
:sadpanda: Off-topic but I wonder if recent versions of TF give us the means to adopt the existing resource by name without a separate import step 🤔 (don't worry about it for now, just thinking out loud here)

Reply:
Hmm, I wonder if using an `import` block here would work, since they're idempotent. Something like:

```hcl
import {
  id = "kube-system/aws-auth"
  to = kubernetes_config_map.aws_auth
}
```

Reply:
TF seems to be happy with the import block, so I've added it and removed that step from the docs

Reply:
Awesome!